examples/cli/discovery.md

```diff
@@ -49,34 +49,14 @@ You can see that the model is discovered, and some stats are printed, but no bui
 
 > See the [Analyze Only documentation](https://github.com/groq/mlagility/blob/main/tools_user_guide.md#analyze-only) for more details.
 
-## Search Directory
+## Benchmark Multiple Scripts
```

Comment:
fix links to this name in cache.md

in general I suggest being super hesitant/careful when renaming markdown headings.